### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#272] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 131746](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8d1299fe-dca5-4d63-968a-e5b20a6555bb)

**Comment by Elia-Belli** [comment]
$U\oplus W\iff (U\cap W)=\underline 0\iff Cx=0$ ha il vettore nullo come unica soluzione, procediamo quindi a risolvere il sistema con matrice associata $C$:

$$\begin{pmatrix}1 & 0 & -1 & 2\\
-2 & 1 & 1& 0\\
0 & 1& 1& 0\\
0 & -1& 0& 4\end{pmatrix}\rightarrow^{Gauss}\begin{pmatrix}1 & 0 & -1 & 2\\
0 & 1 & -1& 0\\
0 & 0& -1& 4\\
0 & 0& 0& 8\end{pmatrix}$$  

Che ha appunto soluzione unica nulla, quindi $(U\cap W)=\underline 0$

Osserviamo invece che $A,B$ sono delle matrici non quadrate i cui sistemi corrispondenti dipendono da due parametri, infatti:

$$\begin{cases}-2x+y-z=0\\
-y+4w=0\end{cases}\rightarrow \begin{cases}x=2w-\frac{1}{2}z\\
y=4w\end{cases}\ \ \ \ \ \ \ \begin{cases}x'-z'+2w'=0\\
y'+w'=0\end{cases}\rightarrow \begin{cases}x'=z'+2y'\\
w'=-y'\end{cases}$$

Dipendendo da 2 parametri ne segue che hanno entrambi dimensione $2\Rightarrow \mathbb{R}^4=U\oplus W$

**Comment by daromacarlo** [comment]
![Immagine 2023-12-01 153749](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/72df6020-87c7-4133-927c-b8f6b59a4776)
![Immagine 2023-12-01 154109](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/5a78a97b-e9c1-4b7d-b9f7-5d5f04da6588)
![Immagine 2023-12-01 153822](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/9a779b3d-9ce2-4fcd-8501-b9069837bd7b)
![Immagine 2023-12-01 153837](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/0ef30e81-d71a-4776-88bc-e409ab8013cb)
![Immagine 2023-12-01 153926](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/77b9b707-5bfb-4b78-aadd-47f3353ff674)

**Comment by daromacarlo** [comment]
@Elia-Belli l'ultima parte dell'esercizio va dimostrata oppure ogni volta che incontro un sistema lineare che dipende da X parametri posso dire che Dim(sottospazio) = Dim(spazio) - X ?

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5f21ef23-e0de-4e10-9b71-30c3674ef9fd)

**Comment by Elia-Belli** [comment]
> @Elia-Belli l'ultima parte dell'esercizio va dimostrata oppure ogni volta che incontro un sistema lineare che dipende da X parametri posso dire che Dim(sottospazio) = Dim(spazio) - X ?

Quando faremo i sistemi non quadrati risulterà più evidente

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/9e438022-e48d-48e3-b70c-fa0c8cbee06f)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8f9d6e51-d3dc-4b32-8172-bb64191cd2e6)
